controllers/controllers/shared/index.go

```diff
@@ -3,15 +3,30 @@ package shared
 import (
 	"context"
 
+	servicesv1alpha1 "code.cloudfoundry.org/cf-k8s-controllers/controllers/apis/services/v1alpha1"
+
 	networkingv1alpha1 "code.cloudfoundry.org/cf-k8s-controllers/controllers/apis/networking/v1alpha1"
 
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
-const DestinationAppName = "destinationAppName"
+const (
+	DestinationAppName         = "destinationAppName"
+	ServiceBindingAppGUIDIndex = "serviceBindingAppGUID"
+)
```

Comment:
It looks like you're introducing a new naming convention for these constants with the `Index` suffix. 2 things about this:

1. For consistency, I'd suggest renaming the existing constant `DestinationAppName` to match the same convention
2. A nice standard practice I've seen in Go with constants is to prefix them all with the same "type" info (e.g `http.StatusOK`, `http.StatusCreated`, etc). Maybe `IndexServiceBindingAppGUID` or `IndexForServiceBindingAppGUID` would be clearer and make it easier to find index constants generally?

Reply:
Done!
Changed to `IndexServiceBindingAppGUID` and `IndexRouteDestinationAppName`